### ISSUE TITLE
Power Management - I'm getting around 8-9 hours.

### ISSUE THREAD

**Issue by Saigoxx** [Edoardo001/Matebook-13-Hackintosh#15]
Maybe borrow something here for us? For example battery life. https://github.com/gnodipac886/MatebookXPro-hackintosh
Power Management - I'm getting around 8-9 hours.

now we have a maximum of 2-3 hours

**Comment by Edoardo001** [comment]
Join telegram group(link on guide page) to test all togheter and help testing if you want.

**Comment by h1ghrise** [comment]
any update on the power management? As @Saigoxx mentioned, its horrible :( (I also get only 2-3 hours)

**Comment by Edoardo001** [comment]
I have better battery life than windows, we have different battery from matebook X

**Comment by Balzac-Guan** [comment]
A proper cpu friend configuration may help extending your battery life to about 4H. For further improvement，you may need to adjust the voltage of your cpu package by *voltageshift*. It could bring your battery life up to about 7H (without MX150 Card) while inappropriate adjustments may damage your laptop.(Sorry for my terrible English, I hope that you could understand what I have typed above)

**Comment by Edoardo001** [comment]
@Lord-Balzac I'm using voltageshift since Mojave. I've undervolted cpu and integrated gpu of 120 and 90mv and disabled turbo boost. I've also done a script to disable turbo or enable  it in one click, so people can enable or disable when they need(using voltageshift). Can't do a script to undervolt because every cpu has his own capabilities. You can find it on telegram group.

Ps: undervolt could made instabilities, but no damages, returning to stock values will restore stability. Overvolt can make damages, but until we are dealing with laptoos, specially ultrabooks, I think we not need to consider it 😂

Cpu friend it's tweaked to lowest frequency laptop can handle(1ghz), lowest cpu can handle is 800mhz, but makes ui laggy.

**Comment by Balzac-Guan** [comment]
> @Lord-Balzac I'm using voltageshift since Mojave. I've undervolted cpu and integrated gpu of 120 and 90mv and disabled turbo boost. I've also done a script to disable turbo or enable it in one click, so people can enable or disable when they need(using voltageshift). Can't do a script to undervolt because every cpu has his own capabilities. You can find it on telegram group.
> 
> Ps: undervolt could made instabilities, but no damages, returning to stock values will restore stability. Overvolt can make damages, but until we are dealing with laptoos, specially ultrabooks, I think we not need to consider it 😂
> 
> Cpu friend it's tweaked to lowest frequency laptop can handle(1ghz), lowest cpu can handle is 800mhz, but makes ui laggy.

I can't agree you more. Ultrabooks designed for mobile working are expected for longer battery life. However the performance schedule which macOS uses is so radical that the phenomenon of the laggy ui may be not apparent（It acts like this on my laptop😂）BTW, it's a pity that the network proxy I currently use does not support telegram now, I think building the configuration personally with parameters tested with XTU is not that difficult😀

**Comment by h1ghrise** [comment]
thanks a lot for your replies :) - Are these settings are already in the build? Or is there some manual Post-install tweaking necessary?

**Comment by Edoardo001** [comment]
@h1ghrise the cpu friend Is done in the build. For the undervolt you have to do yourself, it's easy, just search for voltageshift on Google, and use my version on telegram, because latest have some problem, I've found that patched one by hour of searching. Remember to disable sip first .

@Lord-Balzac yes man, it's easy and the results are pretty good

**Comment by skam2420** [comment]
Hello, I have Matebook 13 2019 (I5 & MX250), Working really good with Catalina, but I only get MAX 6 hours using TurboBoost switcher to disable (Actually works because when its on, the temperatures is high because OC)

**Comment by Edoardo001** [comment]
> Hello, I have Matebook 13 2019 (I5 & MX250), Working really good with Catalina, but I only get MAX 6 hours using TurboBoost switcher to disable (Actually works because when its on, the temperatures is high because OC)

You get better on windows?

**Comment by skam2420** [comment]
> > Hello, I have Matebook 13 2019 (I5 & MX250), Working really good with Catalina, but I only get MAX 6 hours using TurboBoost switcher to disable (Actually works because when its on, the temperatures is high because OC)
> 
> You get better on windows?

No, same time, but the thing is, one of my friend has the same Matebook 2019 but Intel Only and he gets 8-10 hours, My Nvidia is off, MAC, LINUX OR WINDOWS, but the time never can be longer than 6 hours, if I turn the Nvidia ON, the time its about 4 hours

**Comment by Edoardo001** [comment]
> > > Hello, I have Matebook 13 2019 (I5 & MX250), Working really good with Catalina, but I only get MAX 6 hours using TurboBoost switcher to disable (Actually works because when its on, the temperatures is high because OC)
> > 
> > You get better on windows?
> 
> No, same time, but the thing is, one of my friend has the same Matebook 2019 but Intel Only and he gets 8-10 hours, My Nvidia is off, MAC, LINUX OR WINDOWS, but the time never can be longer than 6 hours, if I turn the Nvidia ON, the time its about 4 hours

You have answered your own question. Your friend laptop is for sure a different model, like matebook pro. My is a 2019 matebook 13 and the only way to get 8-9 hours is with AC adapter on 😂
